### ISSUE TITLE
[Request] Telegram Bot Integration

### ISSUE THREAD

**Issue by InoSiX** [mikulash/Octoprint-detector2#11]
Hiya!

Firstly, thanks for the plugin. I look forward to using it. It's efforts from the community, exactly like yours, that really makes Octoprint the single best upgrade for a 3D printer.

Email is a great start for this application. It's certainly a proof of concept. I find email to be slow, however. Perhaps not in terms of actual delivery but more so reviewing and responding to the information (the point after all is to not continue a failed print and save the plastic/energy).

I have been using the Telegram bot plugin to communicate and get updates from my printers in real time (with photo). This is a really neat integration. If possible, it would be great if your plugin handled multiple communication methods. Perhaps email works for some folks and should be available, specially if they don't want to use Telegram. But getting a real time notification to your phone from Telegram is a lot more likely to catch a user's immediate attention, prompting response.

Regardless of implementation, thank you for putting this together.

**Comment by lgallard** [comment]
I agree with @InoSiX, having the Telegram integration would be a perfect match!
